docs/fsharp-core-notes.md

```diff
@@ -1,126 +1,118 @@
+As an F# package author, you also need to make this decision with respect to FSharp.Core:
 
-* Scripts are not projects, just `.fsx` files, possibly referring to other files using `#load` and Libraries using `#r`
+* Targeting an earlier version of FSharp.Core increases your reach because older codebases can use it without issue
+* Targeting a newer version of FSharp.Core lets you use and extend newer features
 
-### Do *not* bundle FSharp.Core with a library 
+This decision is critical, because it can have a network effect. If you choose a higher FSharp.Core version, then that also becomes a dependency for any other package that may depend on your package.
 
-Do _not_ include a copy of FSharp.Core with your library or package.  If you do, you will create havoc for users of
-your library.
+### Package authors should pin their FSharp.Core reference
 
-The decision about which `FSharp.Core` a library binds to is up to the application hosting of the library.
-The library and/or library package can place constraints on this, but it doesn't decide it.
+The default templates for F# projects carry an implicit reference to FSharp.Core. This is ideal for application developers, since applications almost always want to be referencing the highest FSharp.Core available to them. As you upgrade your .NET SDK, the FSharp.Core package referenced implicitly will also be upgraded over time, since FSharp.Core is also distributed with the .NET SDK.
 
-Especially, do _not_ include FSharp.Core in the ``lib`` folder of a NuGet package.
+However, as a package author this means that unless you reference FSharp.Core explicitly, you will default to the latest possible version and thus eliminate any hope of reaching older projects in older environments.
 
-### Always deploy FSharp.Core as part of a compiled application
+### How to explicitly reference FSharp.Core
 
-For applications, FSharp.Core is normally part of the application itself (so-called "xcopy deploy" of FSharp.Core).  
+It's a simple gesture in your project file that pins to FSharp.Core 4.7.2:
 
-For modern templates, this is the default. For older templates, you may need to use ``<Private>true</Private>`` in your project file. In  Visual Studio this is equivalent to setting the `CopyLocal` property to `true` properties for the `FSharp.Core` reference.
+```xml
+<ItemGroup>
+  <PackageReference Update="FSharp.Core" Version="4.7.2" />
+</ItemGroup>
+```
 
-FSharp.Core.dll will normally appear in the `bin` output folder for your application. For example:
+Or if you're using Paket:
 
 ```
-    Directory of ...\ConsoleApplication3\bin\Debug\net5.0
-    
-    18/04/2020  13:20             5,632 ConsoleApplication3.exe
-    14/10/2020  12:12         1,400,472 FSharp.Core.dll
+nuget FSharp.Core >= 4.7.2
 ```
 
-### Always reference FSharp.Core via the NuGet package.
+You'll also want to set `<DisableImplicitFSharpCoreReference>true</DisableImplicitFSharpCoreReference>` if you are using Paket.
```

Comment:
Paket should set this in it's restore.targets for you already, so you shouldn't need to mention it here.

Reply:
oh joy! Will remove. This was annoying to do in the past